Day1.md

```diff
@@ -0,0 +1,5 @@
+# Day 1
+
+Learnt a new language -- JQuery.  Totally different from Javascript.  I like the way how it can change/add things in html.  Seems it is easier than using javascript.  For the responsive part, I didnt realize that the webpages that I usually visit  are actually containing responsive element.  I feel like I have learnt a lot.
```

Comment:
Learned not learnt.  Jquery is not a language, it is a library built with the JavaScript language.